Polyfills/XMLHttpRequest/Source/XMLHttpRequest.cpp

```diff
@@ -161,10 +174,10 @@ namespace Babylon::Polyfills::Internal
 
     void XMLHttpRequest::AddEventListener(const Napi::CallbackInfo& info)
     {
-        std::string eventType = info[0].As<Napi::String>().Utf8Value();
-        Napi::Function eventHandler = info[1].As<Napi::Function>();
+        const std::string eventType{info[0].As<Napi::String>().Utf8Value()};
```

Comment:
I've been following this guideline for brace initialization. We don't just want to blanket replace all with brace initialization. https://stackoverflow.com/a/9977536. So, in this case, it should not be a brace initialization and also pretty much everywhere changed in this file.

Reply:
Fixed